src/types.ts

```diff
@@ -7,18 +7,26 @@ export type SUPPORTED_NETWORKS =
   | "ethereum"
   | "opbnbtestnet"
   | "opbnbmainnet"
-  | "zksyncsepolia";
+  | "zksyncsepolia"
+  | "zksyncmainnet"
+  | "opsepolia"
+  | "opmainnet";
 
 export type REMOTE_NETWORKS =
   | "sepolia"
   | "ethereum"
   | "opbnbtestnet"
   | "opbnbmainnet"
   | "arbitrumsepolia"
-  | "arbitrumone";
+  | "arbitrumone"
+  | "zksyncsepolia"
+  | "zksyncmainnet"
+  | "opsepolia"
+  | "opmainnet";
```

Comment:
Can we do `export type REMOTE_NETWORKS = Exclude<SUPPORTED_NETWORKS, "bscmainnet" | "bsctestnet">`?

Reply:
@GitGuru7

Reply:
https://github.com/VenusProtocol/vips/pull/384/commits/a35c9c4fe7d62b364cd288f07d074869d7ab0e40